Lib/calendar.py

```diff
@@ -275,7 +304,7 @@ def yeardays2calendar(self, year, width=3):
         """
         months = [
             self.monthdays2calendar(year, i)
-            for i in range(January, January+12)
+            for i in range(JANUARY,JANUARY+12)
```

Comment:
```suggestion
            for i in range(JANUARY, JANUARY+12)
```
I think you're accidentally made this